src/language/conditionals.md

```diff
@@ -10,30 +10,39 @@ Pug's first-class conditional syntax allows for optional parentheses.
 
 If you’re coming from Pug v1, you may now omit the leading `-`. Otherwise, it's identical (just regular JavaScript):
 
+You can use strict or loose equality (triple and double equals), as well as any valid expression for a JavaScript `if` statement
+
 ```pug-preview
-- var user = { description: 'foo bar baz' }
-- var authorised = false
+- 
+  var user = {
+    37: { name: "Finn", description: "The Human", authorised: true },
+    38: { name: "Marceline", description: "The Vampire Queen", authorised: false }
+  }
+  var id = 38
 #user
-  if user.description
+  if user[id].description
     h2.green Description
-    p.description= user.description
-  else if authorised
+    p.description= user[id].description
+  if user[id].name === "Finn"
+    h1.yellow Name
+    p.name= user[id].name
+  else if user[id].authorised
     h2.blue Description
     p.description.
       User has no description,
       why not add one...
   else
     h2.red Description
-    p.description User has no description
+    p.description User is not Finn.
```

Comment:
Would this be clearer as:

```pug
  if user[id].name === "Finn"
    h1.yellow Name
    p.name= user[id].name
  else
    h2.red Description
    p.description User is not Finn.

  if user[id].description
    h2.green Description
    p.description= user[id].description
  else
    h2.blue Description
    p.description.
      User has no description,
      why not add one...
```

i.e. first, are they "Finn", second, do they have a description, with a space between the two if blocks.

Reply:
The first section makes sense, I've changed it. 

The second section, I was trying to work in the boolean style 'if exists' check on the more sensible `authorised` element.